scala/scala-impl/resources/META-INF/scala-plugin-common.xml

```diff
@@ -72,7 +72,7 @@
         <testFinder implementation="org.jetbrains.plugins.scala.testingSupport.ScalaTestFinder"/>
         <exceptionFilter implementation="org.jetbrains.plugins.scala.testingSupport.util.scalatest.ScalaTestFailureLocationFilterFactory"/>
         <exceptionFilter implementation="org.jetbrains.plugins.scala.stacktrace.ScalaPackageObjectFilterFactory" order="first"/>
-        <compileServer.plugin classpath="scala-library.jar;compiler-shared.jar;jps/nailgun.jar;jps/compiler-jps.jar;jps/sbt-interface.jar;jps/incremental-compiler.jar"/>
+        <compileServer.plugin classpath="scala-library.jar;compiler-shared.jar;jps/nailgun.jar;jps/compiler-jps.jar;jps/hydra-compiler-jps.jar;jps/sbt-interface.jar;jps/incremental-compiler.jar"/>
```

Comment:
You may register your own `compileServer.plugin` in `hydra-plugin-common.xml` and add `hydra-compiler-jps.jar` there.

You will probably use `sbt-idea-plugin` for plugin setup. It will download scala plugin to some directory, so you will need to find and add `compiler-jps.jar` to dependencies of `hydra-compiler-jps` project in your build file. 
https://github.com/JetBrains/sbt-idea-plugin#ideaexternalplugins--settingkeyideaplugin

@mutcianm can help you with it.

Reply:
Sounds good. I guess we will do this when moving the whole hydra support into its own plugin, which is the next thing I'm working on. But we can merge this PR and then I'll have a new PR that removes all Hydra code from the intellij-scala codebase. Ok?

Reply:
Oh, ok, awesome!